Makefile

```diff
@@ -1,6 +1,11 @@
 .PHONY: all init format lint build build_frontend install_frontend run_frontend run_backend dev help tests coverage
 
 all: help
+VERSION=$(shell grep "^version" pyproject.toml | sed 's/.*\"\(.*\)\"$$/\1/')
```

Comment:
Is this command trying to get a version number?  I ran the command it returns `version = ` too. Is this expected?
```
$ grep "^version" pyproject.toml | sed 's/.*\"\(.*\)\"$$/\1/'
version = "1.0.5"
```

Reply:
Hi, @zzzming! You got this result because you didn't remove one of the $ signs needed to escape sed command inside the makefile